src/main/clm_instMod.F90

```diff
@@ -187,9 +188,13 @@ subroutine clm_instInit(bounds)
     use SoilWaterRetentionCurveFactoryMod  , only : create_soil_water_retention_curve
     use decompMod                          , only : get_proc_bounds
     use BalanceCheckMod                    , only : GetBalanceCheckSkipSteps
+    use FATESFireNoDataMod, only: fates_fire_no_data_type
+    use FATESFireDataMod, only: fates_fire_data_type
     !
     ! !ARGUMENTS    
     type(bounds_type), intent(in) :: bounds  ! processor bounds
+    type(fates_fire_data_type) :: fates_fire_data_inst
```

Comment:
Same here, you just need to know the generic type rather than the specific one

Reply:
Same [comment](https://github.com/ESCOMP/CTSM/pull/991#discussion_r415423650)